PowerEditor/installer/nativeLang/Malayalam.xml

```diff
@@ -0,0 +1,954 @@
+                    <Item id="25014" name="ഡെലിമീറ്ററിന്റെ 7"/>
+                    <Item id="25015" name="ഡെലിമീറ്ററിന്റെ 8"/>
+                    <Item id="25018" name="കീവേഡ് 1"/>
+                    <Item id="25019" name="കീവേഡ്  2"/>
+                    <Item id="25020" name="കീവേഡ്  3"/>
+                    <Item id="25021" name="കീവേഡ്  4"/>
+                    <Item id="25022" name="കീവേഡ്  5"/>
+                    <Item id="25023" name="കീവേഡ്  6"/>
+                    <Item id="25024" name="കീവേഡ്  7"/>
+                    <Item id="25025" name="കീവേഡ്  8"/>
+                    <Item id="25016" name="അഭിപ്രായം"/>
+                    <Item id="25017" name="അഭിപ്രായം ലൈൻ"/>
+                    <Item id="25026" name="ഓപ്പറേറ്റർ 1"/>
+                    <Item id="25027" name="ഓപ്പറേറ്റർ 2"/>
+                    <Item id="25028" name="സംഖ്യാപുസ്തകം"/>
+					<Item id="1" name="ശരി"/>
+                    <Item id="2" name="റദ്ദാക്കുക"/>
+                </StylerDialog>
+                <Folder title=" &amp;&amp;ഫോൾഡർ സ്വതേ">
+                    <Item id="21101" name="സ്ഥിരസ്ഥിതി സ്റ്റൈൽ"/>
+                    <Item id="21102" name="സ്ടിലെർ"/>
+                    <Item id="21105" name="ഡോക്യുമെന്റേഷൻ:"/>
+                    <Item id="21104" name="താൽക്കാലികമായ ഡോക് സൈറ്റ്:"/>
+                    <Item id="21106" name="(വളരെ ശൂന്യമായ വരികൾ മടക്കിക്കളയുന്നു) കോംപാക്ട് മടക്കുക"/>
+                    <Item id="21220" name="കോഡ് 1 രീതിയിൽ നിദ്ര"/>
+                    <Item id="21224" name="തുറക്കുക:"/>
+                    <Item id="21225" name="മിഡിൽ:"/>
+                    <Item id="21226" name="അടയ്ക്കുക:"/>
+                    <Item id="21227" name="സ്ടിലെർ"/>
+                    <Item id="21320" name="കോഡ് 2 രീതിയിൽ നിദ്ര (വിഭാജികൾ needed):"/>
+                    <Item id="21324" name="തുറക്കുക:"/>
+                    <Item id="21325" name="മിഡിൽ:"/>
+                    <Item id="21326" name="അടയ്ക്കുക:"/>
+                    <Item id="21327" name="സ്ടിലെർ"/>
+                    <Item id="21420" name="അഭിപ്രായം രീതിയിൽ നിദ്ര"/>
+                    <Item id="21424" name="തുറക്കുക:"/>
+                    <Item id="21425" name="മിഡിൽ:"/>
+                    <Item id="21426" name="അടയ്ക്കുക:"/>
+                    <Item id="21427" name="സ്ടിലെർ"/>
+                </Folder>
+                <Keywords title="അടയാളവാക്കുകൾ ലിസ്റ്റുകൾ">
+                    <Item id="22101" name="1st ഗ്രൂപ്പ്"/>
+                    <Item id="22201" name="2nd ഗ്രൂപ്പ്"/>
+                    <Item id="22301" name="3rd ഗ്രൂപ്പ്"/>
+                    <Item id="22401" name="4th ഗ്രൂപ്പ്"/>
+                    <Item id="22451" name="5th ഗ്രൂപ്പ്"/>
+                    <Item id="22501" name="6th ഗ്രൂപ്പ്"/>
+                    <Item id="22551" name="7th ഗ്രൂപ്പ്"/>
+                    <Item id="22601" name="8th ഗ്രൂപ്പ്"/>
+                    <Item id="22121" name="പ്രിഫിക്സ് മോഡ്"/>

```

Comment:
Check this. I think it cann't fit on UI, you have to shorten the text length. From 22121 to 22621.

Reply:
@SinghRajenM  Translate text is big we can't do anything for this

Reply:
But showing truncated UI is not useful at all. Then use English string only and mark this in translation file. so that any other person will not update in future as well.